### PR TITLE
More doc stuff

### PR DESCRIPTION
**Pull request by pllim**
Separated installation section into subsections to de-clutter index page. Added stub for IPython Notebook instructions (attn: @eteq). Completed API doc. Corrected typo and streamlined text formatting.

### CODE REVIEW

File: stginga/nbinteract.py [hunk 1]
```diff
@@ -15,10 +15,44 @@
 from ginga.AstroImage import AstroImage
 from ginga.web.pgw import Widgets, js, PgHelp, ipg
 
-__all__ = ['start_server', 'GingaServer', '']
+__all__ = ['GingaServer']
```
Comment (pllim) [inline stginga/nbinteract.py:18]: @eteq , looks like a typo to me, but can you confirm this?

File: stginga/plugin_info.py [hunk 2]
```diff
@@ -102,7 +49,5 @@ def _get_stginga_plugins():
 
 
 def show_plugin_install_info():
-    """
-    Prints the documentation on how to install the ginga plugins.
-    """
-    print(__doc__)
+    """Print the documentation on how to install the ginga plugins."""
+    print('See http://stginga.readthedocs.org/en/latest/run.html')
```
Comment (pllim) [inline stginga/plugin_info.py:53]: @eteq , I am not very sure if this function is even needed anymore. Thoughts?

**Comment by pllim** [comment]
@eteq , I'll merge this by COB (Eastern time) today if I don't hear from you. We can always open another issue to follow-up if needed. FYI.

File: stginga/nbinteract.py (hunk 1 above)
Comment (eteq) [inline stginga/nbinteract.py:18]: yep, `start_server` is a holdover from an earlier way I had this structured.  So the change you made is correct.

**Comment by eteq** [comment]
@pllim - looks good to me!

**Comment by pllim** [comment]
Thanks, @eteq :+1:

**Pull request merged**